src/multipleHistoricalSessions/indicator/historicalSessionIndicator.vue

```diff
@@ -48,64 +62,71 @@
 <script>
 import HistoricalSessionSelector from '../sessionSelector/historicalSessionSelector.vue';
 import SessionService from 'services/session/SessionService';
-
+import { formatMultipleSessionNumbers } from '../../utils/strings';
 export default {
-  inject: ['openmct', 'table'],
-  components: {
-    HistoricalSessionSelector
-  },
-  computed: {
-    filteredByMessageString() {
-      let sessionOrSessions;
-
-      if (this.sessionFilter.numbers.length === 1) {
-        sessionOrSessions = 'session';
-      } else {
-        sessionOrSessions = 'sessions';
-      }
-      return `Historical queries filtered by ${this.sessionFilter.numbers.length} ${sessionOrSessions}`;
-    }
-  },
-  data() {
-    return {
-      sessionFilter: {},
-      numFilteredSessions: 8,
-      availableSessions: [],
-      showSessionSelector: false,
-      isRequestingSessions: false,
-      historicalSessionFilterDisabled: false
-    };
-  },
-  methods: {
-    setHistoricalSessionFilter(sessions) {
-      if (sessions) {
-        this.sessionFilter = sessions;
-      } else {
-        this.sessionFilter = {};
-      }
-    },
-    setAvailableSessions(sessions) {
-      this.isRequestingSessions = false;
-      this.availableSessions = sessions;
+    inject: [
+        'openmct',
+        'table'
+    ],
+    components: {
+        HistoricalSessionSelector
     },
-    openSessionSelector() {
-      this.showSessionSelector = true;
+    computed: {
+        filteredByMessageString() {
+            let sessionOrSessions;            
+
+            if (this.sessionFilter.numbers.length === 1) {
+                sessionOrSessions = `session: ${this.sessionFilter.numbers[0]}`;
+            } else {
+                sessionOrSessions = 'sessions';
+            }
+            return `Historical queries filtered by ${this.sessionFilter.numbers.length} ${sessionOrSessions}`;
+        },
+        filteredByTitleString() {
+            let sessionNumbers = formatMultipleSessionNumbers(this.sessionFilter.numbers);
+
+            return `Currently filtering on host ${this.sessionFilter.host} by: ${sessionNumbers}`;
```

Comment:
suggest replacing ` on host ${this.sessionFilter.host}` with ${this.sessionFilter.host !== undefined ? ` on ${this.sessionFilter.host}` : ''}`. Oof, nested template literals... first time for everything.

Reply:
gotcha AND avoided the nested template literal